src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -144,7 +144,7 @@ private void EnableTimer(float pollingIntervalInSeconds)
                     _nextPollingTimeStamp = DateTime.UtcNow + new TimeSpan(0, 0, (int)pollingIntervalInSeconds);
 
                     // Create the polling thread and init all the shared state if needed
-                    if (s_pollingThread == null)
+                    if (s_pollingThread == null && !OperatingSystem.IsBrowser())
```

Comment:
Most likely the method should have `[UnsupportedOSPlatform("browser")]` instead but that would cause the caller to also have `[UnsupportedOSPlatform("browser")]` and eventually entire type would need to have `[UnsupportedOSPlatform("browser")]`, which would cause warnings in DiagnosticCounter.cs which would affect all inherited counters. So that might not the correct solution too, please let me know what we should do here

Reply:
If this is disabled, the whole class is broken.  Either the whole thing needs to be unsupported, along with all of the diagnostic counters, or mono needs a custom implementation for all of these for browser.